articles/quantum-InstallConfig.md

```diff
@@ -33,34 +30,54 @@ ms.topic: article
+3. _**Important!**_ &nbsp;When you are presented with the option to select the tools for specific workloads, check the box for **.NET Core cross-platform development**.
+   If there are any other workloads you would like to install, you can select them as well at this step.
 4. After selecting your workloads, click **Install** to complete the installation.
+5. Once you have installed Visual Studio 2017, download the [Microsoft Quantum Development Kit for Visual Studio](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit) extension from the Visual Studio Marketplace.
+   Double-click the VSIX once it is downloaded to install the extension.
+
+### [Installing Visual Studio Code and the .NET Core SDK (Windows, macOS, Linux)](#tab/tabid-vscode)
 
-#### [Installing Visual Studio Code (Windows, macOS, Linux)](#tab/tabid-vscode)
+The Quantum Development Kit uses the .NET Core SDK (2.0 or later) to make it easy to create, build, and run Q# projects from the command line.
+We recommend using the Quantum Development Kit together with Visual Studio Code.
 
-You can download Visual Studio Code for free for your platform.
+1. Please see [Getting started with .NET](https://www.microsoft.com/net/learn/get-started) for how to download and install the .NET Core SDK.
+2. Once the .NET Core SDK is installed, run the following command in your favorite command line (e.g.: PowerShell or Bash) to download the latest templates for creating new Q# applications and libraries:
+   ```bash
+   $ dotnet new -i "Microsoft.Quantum.ProjectTemplates::0.2.1806.3001-preview"
+   ```
 1. Go to the [Visual Studio Code website](https://code.visualstudio.com/).
 2. Pick the download for your platform and follow the installation prompts.
+    > [!NOTE]
+    > When you run Visual Studio Code for the first time, you may be prompted to install Git if you have not already done so.
+    > Git is not required to use the Quantum Development Kit, but works great for managing and sharing Q# projects, and for contributing back to the Q# libraries and samples.
+4. Once Visual Studio Code is installed, go to the [Microsoft Quantum Development Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode) extension on the Visual Studio Marketplace and press Install.
+
+> [!TIP]
+> You can also install the Quantum Development Kit extension for Visual Studio Code from the *Go to file...* palette.
+> Press **Ctrl+P** or **⌘+P** from within Visual Studio Code, type or paste `ext install quantum.quantum-devkit-vscode`, and press **Enter**.
 
 ****
 
-## Installing and Validating the Q# Development Environment
+You should now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
+The next section will cover how to build the samples and standard libraries provided along with the Quantum Development Kit.
```

Comment:
Is this phrase redundant? It just repeats the next section tile

Reply:
I think it helps the prose read better across that transition, but I can reword it, then.

Reply:
I still don't like it... Together with the header and the first sentence of the next paragraph it's repeated three times.